src/App.jsx

```diff
@@ -11,7 +12,6 @@ const article = {
 export const App = () => (
   <div className="App">
     <h1>React Article</h1>
-
-    {/* Add an <Article /> and pass `title`, `text` and `date` from the `article` object as props */}
+    <Article text={article.text} title={article.title} date={article.date} />
```

Comment:
```suggestion
    <Article
      text={article.text}
      title={article.title}
      date={article.date}
    />
```

now it is more pretty 🤩